.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
```

Comment:
@castastrophe mind double-checking my work here? I want to be sure I'm not missing any setup steps.